pkg/resourceinterpreter/defaultinterpreter/prune/prune_test.go

```diff
@@ -0,0 +1,232 @@
+package prune
+
+import (
+	"fmt"
+	"strings"
+	"testing"
+
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+
+	"github.com/karmada-io/karmada/pkg/util"
+)
+
+type field []string
+
+func (f field) String() string {
+	return "." + strings.Join(f, ".")
+}
+
+type Test struct {
```

Comment:
This structure seems does not need to be exported.

Reply:
> This structure seems does not need to be exported.

Thanks for your suggestion, I ignored this.